lib/linter/linter.js

```diff
@@ -77,13 +78,15 @@ const globals = require("../../conf/globals");
  * @property {number} line The line number
  * @property {number} column The column number
  * @property {(string|null)} ruleId The rule ID
+ * @property {string} justification The justification of directive
  */
 
 /**
  * The private data for `Linter` instance.
  * @typedef {Object} LinterInternalSlots
  * @property {ConfigArray|null} lastConfigArray The `ConfigArray` instance that the last `verify()` call used.
  * @property {SourceCode|null} lastSourceCode The `SourceCode` instance that the last `verify()` call used.
+ * @property {SuppressedLintMessage[]|null} lastSuppressedMessages The `SuppressedLintMessage[]` instance that the last `verify()` call produced.
```

Comment:
```suggestion
 * @property {SuppressedLintMessage[]} lastSuppressedMessages The `SuppressedLintMessage[]` instance that the last `verify()` call produced.
```

`lastSuppressedMessages` can never be `null`?